Corona-Warn-App/src/main/java/de/rki/coronawarnapp/dccticketing/core/server/DccTicketingApiV1.kt

```diff
@@ -19,5 +19,17 @@ interface DccTicketingApiV1 {
         @Url url: String,
         @Header("Authorization") authorizationHeader: String,
         @Body body: AccessTokenRequest
-    ): Response<String>
+    ): Response
```

Comment:
@jurajkusnier  I changed this to return okhttp response based  on @BMItr  feedback

Reply:
`retrofit2.Response` encapsulate `okhttp3.Response` together with data formatted data from the server. I already have mentioned how to use it for cert. pinning in the TODO. So I am not sure what's really improved here.
https://github.com/corona-warn-app/cwa-app-android/blob/2c2ea65709833e53e47719a2c9dd712cf61b19a3/Corona-Warn-App/src/main/java/de/rki/coronawarnapp/dccticketing/core/server/DccTicketingServer.kt#L100

Reply:
I think what Bernd meant that if the response is `String` it won't be simply read and wrapped in Retrofit response , you have to read the response body , Did you try it ?

Reply:
I meant: Response<String> -> Response<**ResponseBody**>